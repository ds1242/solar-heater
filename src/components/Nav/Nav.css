nav {
    background-color: var(--quartiary_color);
}

h1 {
    color:var(--secondary_color);
    text-decoration:none;
}
h1:hover {
    color:var(--primary_color);
    text-decoration:none;
}