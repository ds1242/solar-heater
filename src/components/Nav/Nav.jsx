import React from 'react';
import { Link } from 'react-router-dom'
import './Nav.css';

function Nav() {
    return (
        <nav className='navbar navbar-expand-sm'>
            <div className='container-fluid'>
                {/* <Link to={'/'} className='navbar-brand'> */}
                    <h1>Solar Heater</h1>
                {/* </Link> */}
                
            </div>
        </nav>
    )
};


export default Nav;