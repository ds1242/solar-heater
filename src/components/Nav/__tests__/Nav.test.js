import React from 'react';
import { render, cleanup } from '@testing-library/react';
import '@testing-library/jest-dom/extend-expect';
import Nav from '../Nav';

afterAll(cleanup);

describe('About component', () => {
    it('renders', () => {
        render(<Nav />);
    });

    it('matches snapshot DOM node structure', () => {
        const { asFragment } = render(<Nav />);

        expect(asFragment()).toMatchSnapshot();
    });
});