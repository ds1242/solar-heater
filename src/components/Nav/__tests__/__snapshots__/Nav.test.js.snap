// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`About component matches snapshot DOM node structure 1`] = `
<DocumentFragment>
  <nav
    class="navbar navbar-expand-sm"
  >
    <div
      class="container-fluid"
    >
      <h1>
        Solar Heater
      </h1>
    </div>
  </nav>
</DocumentFragment>
`;
